### PR TITLE
improved Terminal send function

### PR DESCRIPTION
**Pull request by AlphaLima**
remember sent command when dialog is re-opened.
don't add the checksum to the edit text but when sending

### CODE REVIEW

File: Common/Source/Dialogs/dlgTerminal.cpp [hunk 1]
```diff
@@ -203,31 +203,32 @@ TCHAR  szCheck[254];
...
     {
-      DeviceList[active].Com->WriteString(TxText);
+      DeviceList[active].Com->WriteString(Tmp);
       ComCheck_AddText(_T("\r\n > "));
-      ComCheck_AddText(TxText);
+      ComCheck_AddText(Tmp);
     }
     else
-	  _tcsncpy(TxText, MsgToken(2392),190);  // _@M2392_ "Error: Port not opened!"
+	  _tcsncpy(Tmp, MsgToken(2392),190);  // _@M2392_ "Error: Port not opened!"
```
Comment (brunotl) [inline Common/Source/Dialogs/dlgTerminal.cpp:228]: Value set but never used

File: Common/Source/Dialogs/dlgTerminal.cpp [hunk 2]
```diff
@@ -261,6 +262,12 @@ void dlgTerminal(int portnumber) {
   OnPortClicked(NULL);
 
 
+  if(wf)
+  {
+	WndProperty* wp = (WndProperty*)wf->FindByName(TEXT("prpSendText"));
+    wp->SetText(TxText);
+    wp->GetDataField()->Set(TxText);
+  }
```
Comment (brunotl) [inline Common/Source/Dialogs/dlgTerminal.cpp:270]: WndProperty::SetText must be used only if  WndProperty::GetDataField() return nullptr
instead use : 
``` cpp 
wp->GetDataField()->Set(TxText);
wp->RefreshDisplay();
```

File: Common/Source/Dialogs/dlgTerminal.cpp [hunk 3]
```diff
@@ -19,7 +19,7 @@
 static WndForm *wf=NULL;
 static WndListFrame *wTTYList=NULL;
 static WndOwnerDrawFrame *wTTYListEntry = NULL;
-
+static TCHAR TxText[MAX_NMEA_LEN] =_T("$LXDT,GET,MC_BAL");
```
[the remark this answers is not in the document]
Comment (AlphaLima) [reply Common/Source/Dialogs/dlgTerminal.cpp]: done!

**Comment by AlphaLima** [comment]
all done with following commit

**Pull request merged**